File: delightreading-server/src/main/java/com/delightreading/user/Experience.java
package com.delightreading.user;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.databind.PropertyNamingStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.time.LocalDate;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
public class Experience implements Serializable {
    String kind; // education, project, competition, hackathon,
    String institution; // Weekend Hackaton Org.
    String location; // Boston
    String title; // E.g. Participated in 12th Weekend Hackhaton
    String subject; // E.g. Entrepreneurship
    String description;
    String achievements; // 1st place.
    String fromDate; // 20150200 some day in February 2015
    String toDate;
}

File: delightreading-backend/src/main/kotlin/com/delightreading/user/model/Experience.kt
package com.delightreading.user.model

import java.io.Serializable

data class Experience(
    var kind: String, // education, project, competition, hackathon,
    var institution: String? = null, // Weekend Hackathon Org.
    var location: String? = null,// Boston
    var title: String, // E.g. Participated in 12th Weekend Hackathon
    var subject: String? = null, // E.g. Entrepreneurship
    var description: String? = null,
    var achievements: String? = null, // 1st place.
    var fromDate: String? = null, // 20150200 some day in February 2015
    var toDate: String? = null
): Serializable {
}